embedded-keycloak/src/main/java/com/playtika/test/keycloak/KeycloakContainer.java

```diff
@@ -0,0 +1,66 @@
+package com.playtika.test.keycloak;
+
+import static com.playtika.test.common.utils.ContainerUtils.containerLogsConsumer;
+import static java.lang.String.format;
+import static java.time.Duration.ofSeconds;
+
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.MountableFile;
+
+@Slf4j
+public class KeycloakContainer extends GenericContainer<KeycloakContainer> {
+
+    public static final String KEYCLOAK_USER = "admin";
+    public static final String KEYCLOAK_PASSWORD = "letmein";
+    public static final int HTTP = 8080;
+
+    private final KeycloakProperties properties;
+
+    public KeycloakContainer(KeycloakProperties properties) {
+        super(properties.getDockerImage());
+        this.properties = properties;
+    }
+
+    @Override
+    protected void configure() {
+        withEnv("KEYCLOAK_USER", KEYCLOAK_USER);
+        withEnv("KEYCLOAK_PASSWORD", KEYCLOAK_PASSWORD);
+
+        withCommand(
+            "-c standalone.xml",
```

Comment:
According to: https://www.keycloak.org/docs/7.0/server_installation/#standalone-configuration
depedending on the configuration file Keycloak can start in different modes, so it looks like it would be useful to have this property in KeycloakProperties as well